tests/framework/e2e/etcd_process.go

```diff
@@ -179,3 +182,14 @@ func (ep *EtcdServerProcess) Logs() LogsExpect {
 	}
 	return ep.proc
 }
+
+func ExpectLog(t *testing.T, ep EtcdProcess, expectLog string) {
+	t.Helper()
+	var err error
+	testutils.ExecuteWithTimeout(t, 30*time.Second, func() {
+		_, err = ep.Logs().Expect(expectLog)
```

Comment:
`Expect` is deprecated, please use `ExpectWithContext` instead.

Reply:
I don't think I am understanding this request.  I can't find any other uses of `ExpectWithContext` in the etcd repo and google isn't giving me much either

Reply:
`ExpectWithContext` was just added yesterday

Reply:
Ok found it and done now